src/Resources/views/Form/Type/sonata_type_model_autocomplete.html.twig

```diff
@@ -97,7 +97,7 @@ file that was distributed with this source code.
                                 '{{ req_param_name_page_number }}': (params.page !== 'undefined' ? params.page : 1),
 
                                 // admin
-                                {% if sonata_admin.admin is not null %}
+                                {% if sonata_admin.admin %}
```

Comment:
Not sure if we should apply this kind of fix.

Maybe it's a a theoretical problem, but someone could have overriden this template.

It's more safe to revert this line:

https://github.com/sonata-project/SonataAdminBundle/blob/0639d128cb0516584ab435c0a726dd205ded8ef9/src/Resources/views/Form/Type/sonata_type_model_autocomplete.html.twig#L100-L101

Reply:
There is not just the line you reported.

The check in php was 
```
if ($view->parent && $view->parent->vars['sonata_admin_enabled'] && !($sonataAdmin['admin'] ?? false)) {
```
And phpstan were reporting some errors about the `!($sonataAdmin['admin'] ?? false)` which didn't respect the strict plugin.

Moreover, we were already mixing false
https://github.com/VincentLanglet/SonataAdminBundle/blob/3.x/src/Form/Extension/Field/Type/FormTypeFieldExtension.php#L105
And null
https://github.com/VincentLanglet/SonataAdminBundle/blob/3.x/src/Form/Extension/Field/Type/FormTypeFieldExtension.php#L52

So it make sens to change this check in the template. We already have templates with this checks.
(And not checking exclusively for `null`).